docs/recipes/direct-audit/direct-audit.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+
+const makeDevtoolsLog = require('lighthouse/lighthouse-core/test/network-records-to-devtools-log.js');
```

Comment:
warning: this was written to create devtools logs good enough for tests, not necessarily good in and of themselves :)

The intent was to keep improving the generation but I think there's only been a need for that once.